lib/bankaccounts/account.rb

```diff
@@ -0,0 +1,53 @@
+module BankAccounts
+
+  class Account
+    attr_accessor :id, :min_balance, :initial_balance, :balance, :fee
+
+    def initialize(id, initial_balance = 0)
+      @id = id
+      @min_balance = 0
+      @initial_balance = initial_balance
+      # Raises ArgumentError if initial_balance < 0
+      check_initial_balance(initial_balance)
+      @balance = initial_balance
+      @fee = 0
+    end
+
+    def check_initial_balance(initial_balance)
+      if initial_balance == nil || initial_balance < @min_balance
```

Comment:
Good, you checked for nil values :+1: 

Let me show you another way to check for nils in ruby:
```
if initial_balance.nil? || initial_balance < @min_balance
```